pombola/south_africa/views/elections.py

```diff
@@ -6,6 +6,12 @@
 
 from pombola.core import models
 
+def get_candidate_ranking(candidate):
```

Comment:
What do you think of adding the suffix _sort_key which helps explain why it might return ''?

```suggestion
def get_candidate_rank_sort_key(candidate):
```

Reply:
Makes sense :+1: